packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -941,6 +943,16 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  bool isStructuredErrorsEnabled() {
+    return const bool.fromEnvironment('structuredErrors');
+  }
+
+  void _initStructuredErrorHandler() {
+    if (isStructuredErrorsEnabled()) {
```

Comment:
move this if statement directly into initServiceExtensions. it is just setting the initial value for the service extension.